nucypher/policy/policies.py

```diff
@@ -303,17 +406,22 @@ def publish_treasure_map(self, network_middleware: RestMiddleware, blockchain_si
 
         self.alice.block_until_number_of_known_nodes_is(8, timeout=2, learn_on_this_thread=True)
 
-        publication_deferreds = list()
-        for node in self.bob.matching_nodes_among(self.alice.known_nodes):
+        target_nodes = self.bob.matching_nodes_among(self.alice.known_nodes)
+        self.publishing_mutex = NodeEngagementMutex(f=self.put_treasure_map_on_node,
+                                                    nodes=target_nodes,
+                                                    network_middleware=network_middleware,
+                                                    percent_to_complete_before_release=10)
+
+        self.publishing_mutex.start()
```

Comment:
This method purports to return a `NodeEngagementMutex` but we don't return it. Is that still the plan or should the method return `None`?